scss/titles.scss

```diff
@@ -0,0 +1,14 @@
+.page-title {
+  position: relative;
+  left: 70px;
+  top: 50px;
+  &::before {
+    content: url('assets/triangle.svg');
+    height: 90px;
+    width: 70px;
+    bottom: -31px;
+    left: -50px;
```

Comment:
Try to find a possitive `right` and `top` numbers instead of negatives `left`  and `bottom`. If its not possible, ignore this comment